L1Trigger/L1THGCal/plugins/backend/HGCalBackendStage1Processor.cc

```diff
@@ -53,7 +54,7 @@ class HGCalBackendStage1Processor : public HGCalBackendLayer1ProcessorBase {
 //      triggerCellsPtrs.push_back(ptr);
 //    }
      
-    
+    for (const auto& module_fpga : triggerCellsPtrs) {
```

Comment:
The variable name `module_fpga` is confusing as this is a pair of (fpga id, trigger cells), and not a pair of (module id, fpga id)